packages/mobile/package.json

```diff
@@ -1,9 +1,10 @@
 {
-    "name": "mobile-wallet",
+    "name": "firefly-mobile",
+    "productName": "Firefly Mobile",
```

Comment:
I think `productName` is specific to `electron-builder`, so this might not make a difference for Firefly Mobile

Reply:
Okay, I'll revert it just in case.